contents/founders/first-1000-users.md

```diff
@@ -0,0 +1,231 @@
+
+We figured our target audience would appreciate the transparency, and the ability to inspect our code before self-hosting it. We chose a permissive MIT license to see if enterprises would use it – we felt they'd have rules around using non-permissive open-source licenses. We also [wrote a public company handbook](https://posthog.com/handbook).
+
+#### 4. Make it easy to deploy
+
+We created a one-click Heroku deploy for this. We knew this wouldn’t scale, but it was the fastest way to get started. We also built event autocapture into our SDKs, so user’s didn’t have to manually instrument events to get started.
+
+#### 5. Not to monetize at launch
+
+Why? Because none of the multi-billion dollar open-source companies I researched monetized in the first five years. We were also happy with a VC-funded model, so we could afford to do this.
+
+This decision had two important benefits:
+
+1. It enabled us to move quickly. 
+2. It managed early user expectations. 
+
+We didn’t need to build a payment flow, or agonize over how to convert open-source users into paying customers, and people were more forgiving of issues.
+
+This worked for *us*, but it isn't universally good advice. 
+
+We were entering a crowded market with multiple competitors with around $100m in revenue, so we were confident we’d find a way to monetize eventually. 
+
+Had we been building into a new market with less validation from competitors, or bootstrapping, then monetizing sooner would have been important.
+
+### Finding and onboarding our first users
+
+I started by asking friends because:
+
+1. It’s the easiest way to find initial users. 
+2. It’s better to build a real product for a real person than a hypothetical one.
+
+Doing this surfaced lots of edge cases we wouldn’t have considered on our own.
+
+We weren't self-serve at this stage. In fact, we had to create new users by manually editing the database. We set people up over Slack, WhatsApp, or in-person. We managed to get to 10 users mostly from using our personal networks.
+
+Once we were confident friends could use the product and found it useful, we focused on making PostHog self-serve by creating a one-click Heroku deployment. We knew this wasn’t scalable – we were prioritizing validating our idea quickly.
+
+![meetings](https://res.cloudinary.com/dmukukwp6/image/upload/meetings_a65d6cf7dd.png)
+
+Once we were confident in self-serve, I started reaching out to more people I knew, and expanded into some cold outreach via groups I was in, getting introductions, and messaging via LinkedIn.
+
+This got us to a couple dozen users within a few weeks, but the most critical part was figuring out if people we didn’t know would use the product, and keep coming back.
+
+This is where we failed on several previous ideas. If people sign up but don’t come back over and over, you’ve probably found a valid problem, but not solved it. That's either because your product needs reimagining, or it's painful or confusing to use.
```

Comment:
Try to find a way to have one less paragraph in this section, take it or leave it.
```suggestion
Once we were confident in self-serve, I started reaching out to more people I knew, and expanded into some cold outreach via groups I was in, getting introductions, and messaging via LinkedIn. This got us to a couple dozen users within a few weeks. 

The most critical part of this stage was figuring out if people we didn’t know would use the product, and keep coming back. This is where we failed on several previous ideas. If people sign up but don’t come back over and over, you’ve probably found a valid problem, but not solved it. That's either because your product needs reimagining, or it's painful or confusing to use.
```